Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -16,22 +25,70 @@ BatteryStateOfChargeService::~BatteryStateOfChargeService()
 
 double BatteryStateOfChargeService::totalAmpHoursUsed() const
 {
-    return 0.0;
+
+
+     return amphoursUsed;
 }
 
 bool BatteryStateOfChargeService::isCharging() const
 {
-    return false;
+if(current<0)
```

Comment:
Leave a space between operators.
`if(current < 0)`

Reply:
Everything inside braces should be indented one:
````
 bool BatteryStateOfChargeService::isCharging() const
{
    //stuff
}
````
instead of:
````
 bool BatteryStateOfChargeService::isCharging() const
{
//stuff
}
````